benchmarks/_sk_keras.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import keras
+from keras.models import Model
+from keras.layers import Input, Dense, Dropout
+from keras.optimizers import Adam, RMSprop
+from sklearn.metrics import r2_score
+from keras.regularizers import l2
+
+
+class _KerasModel():
```

Comment:
I would remove a generic name such as `_KerasModel` and rename it to `FullyConnectedNetwork` or something similar.